api-spec.md

```diff
@@ -105,15 +105,22 @@ See the [OpenAPI specification document](openapi/STAC.yaml).
 provided search predicates, probably containing search metadata from the `search` extension |
 
 The `/` endpoint should function as a complete `Catalog` representation of all the data contained in the API and linked 
-to in some way from root through `Collections` and `Items`.
+to in some way from root to `Collections` and `Items` through `children` and `child` Links.
```

Comment:
What is a `children` link? Was that meant to be:

```suggestion
to in some way from root to `Collections` and `Items` through `child` and `item` Links.
```

Reply:
I see now that `children` is not one of the Relation types listed in https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md -- maybe it should be? I copied `children` from https://earth-search.aws.element84.com/v0 when updating EOD, where the href is to `/collection`.  @matthewhanson maybe you have a view on this?

Reply:
if we don't add `children`, I would phrase as:

The `/` endpoint should function as a complete `Catalog` representation of all the data contained in the API. This data is linked to from the root by a `search` Link and one or more `child` Links.

Reply:
We have child defined for Catalog/Collections. What's wrong with that?
The reference to /collections is `data` as defined in OAFeat.

Reply:
I wasn't aware that's rel=data in OAFeat.